addons/payment/i18n/payment.pot

```diff
@@ -2213,3 +2213,11 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:payment.transaction_status
 msgid "warning"
 msgstr ""
+
+#. module: payment
+#: code:addons/payment/models/account_journal.py:0
+#, python-format
+msgid ""
+"You must first deactivate a payment acquirer before deleting its journal.\n"
+"Linked acquirer(s): %s"
```

Comment:
This must be updated to allow the string to be translated.

By the way, it should be
```suggestion
"Linked providers: %s"
```